README.md

```diff
@@ -1,27 +1,25 @@
-# bpmn-visualization
+<h1 align="center">BPMN Visualization</h1> <br>
+<p align="center">
+   <img title="BPMN Visualization" src="config/logo.png" width="812" height="170">
```

Comment:
:question: about the image, the config folder seems not accurate for something related to the documention and it could also be displayed in the html documentation
So could we move it elsewhere and also display it in the html documentation